### DISCUSSION TITLE
Connecting to wallet

### DISCUSSION THREAD

**Discussion by jmoe28** [Chia-Network/chia-blockchain#6069] [Support]
I restarted laptop multiple times. When I double click Chia icon, I keep getting "Connecting to wallet" in a endless loop. Does anyone knows the root cause and how to resolve issue?

**Comment by nodaone** [comment]
I have the same [problem (#6049)](https://github.com/Chia-Network/chia-blockchain/discussions/6049)

And someone else as well [here](https://github.com/Chia-Network/chia-blockchain/issues/6076).

**Reply by nodaone** [reply]
I am curious for the solution as well... ;-)

**Reply by keeperlink** [reply]
Same issue

**Reply by keeperlink** [reply]
[That ](https://github.com/Chia-Network/chia-blockchain/issues/6076#issuecomment-850698149) solution helped me, but instead if deleting all, I just restored a week-old backup of the /wallet/db/ directory and it synced in few minutes

**Comment by BrandtH22** [comment]
@jmoe28 >, since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
